apps/api/src/api/members/api.ts

```diff
@@ -327,3 +327,30 @@ export const removeTeamMember = createApiHandler({
     return { success: true };
   },
 });
+
+export const registryMemberActivity = createApiHandler({
```

Comment:
```suggestion
export const registerMemberActivity = createApiHandler({
```
i would rename every ocurrence to "registerMemberActivity". "Registry" means something else